conf/messages.cy

```diff
@@ -142,6 +142,11 @@ paymentsHistory.activeTab = Yn edrych ar hyn o bryd ar hanes taliadau o {0}
 paymentsHistory.inactiveTab = Bwrw golwg dros hanes taliadau o {0}
 paymentsHistory.noPreviousYearHistory = Ni wnaethoch daliadau na chael unrhyw daliadau yn ystod y flwyddyn hon.
 paymentsHistory.noCurrentYearHistory = Nid ydych wedi gwneud taliadau na chael unrhyw daliadau hyd yma yn ystod y flwyddyn hon.
+paymentHistory.previousPayments = Taliadau blaenorol
```

Comment:
Apologies; some of these message keys are wrong (missing the `.previousPayments` part) - I forgot to update the Welsh spreadsheet after the second translation and the original keys were just placeholders I believe. If you look in master you'll see the actual ones.

Reply:
No worries - just updated it.